inequality/perimeter_area_ratio_spatial_dissimilarity.py

```diff
@@ -0,0 +1,148 @@
+"""
+Perimeter/Area Ratio Spatial Dissimilarity based Segregation Metrics
+"""
+
+__author__ = "Renan X. Cortes <renanc@ucr.edu> and Sergio J. Rey <sergio.rey@ucr.edu>"
+
+import numpy as np
+import pandas as pd
+#import libpysal
+from util import _return_length_weighted_w
+from dissimilarity import _dissim
+from sklearn.metrics.pairwise import manhattan_distances
+
+__all__ = ['Perimeter_Area_Ratio_Spatial_Dissim']
+
+
+def _perimeter_area_ratio_spatial_dissim(data, group_pop_var, total_pop_var, std = True):
+    """
+    Calculation of Perimeter/Area Ratio Spatial Dissimilarity index
+
+    Parameters
+    ----------
+
+    data          : a geopandas DataFrame with a 'geometry' column.
+    
+    group_pop_var : string
+                    The name of variable in data that contains the population size of the group of interest
+                    
+    total_pop_var : string
+                    The name of variable in data that contains the total population of the unit
+                    
+    std           : boolean
```

Comment:
call this `standardize`? `std` is pretty vague.

Reply:
Yes! I'll change to 'standardize' just like other builtin PySAL functions.

Reply:
Not as important, but your other variable names are so good and explanatory, this would be better if it matched :smile:

Reply:
No problem!